### PR TITLE
add "titan test" network

### PR DESCRIPTION
**Pull request by nezha90**

### CODE REVIEW

File: cosmos/titan-test.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "low": 0.005
+      }
+    }
+  ],
+  "rest": "http://lcd.titannet.io:1317",
+  "rpc": "http://rpc.titannet.io:26657",
+  "stakeCurrency": {
+    "coinDecimals": 6,
+    "coinDenom": "TTN",
+    "coinGeckoId": "titan",
```
Comment (editaahn) [inline cosmos/titan-test.json:45]: This ID is currently unavailable now on CoinGecko. Please try again later when it becomes accessible. Alternatively, consider removing it, if you want this to be pushed promptly.

File: cosmos/titan-test.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "coinMinimalDenom": "uttn",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/titan-test/uttn.png"
+  },
+   "nodeProvider": {
+    "name": "titan",
+    "email": "yuan@titannet.io"
+  }
```
Comment (editaahn) [inline cosmos/titan-test.json:52]: It's better to include the website, as it provides a more reliable contact point compared to email, which may change.
If `https://titannet.io/` is relative, it seems good to use.

File: cosmos/titan-test.json (hunk 1 above)
Comment (nezha90) [reply cosmos/titan-test.json:45]: Do you mean "titan"? I searched for "TTN" on CoinGecko and there are similar ones. Can I still use it?

Comment (editaahn) [reply cosmos/titan-test.json]: coinGeckoId you provided is unable to access.
https://www.coingecko.com/en/coins/titan

You should remove it if it's not a officially registered coinGeckoId.

Comment (nezha90) [reply cosmos/titan-test.json]: very thankful

**Pull request merged**